ols/app/endpoints/ols.py

```diff
@@ -184,6 +145,64 @@ def conversation_request(
     )
 
 
+def process_request(auth: Any, llm_request: LLMRequest):
+    """Process incoming request."""
+    timestamps = {"start": time.time()}
```

Comment:
This change will open the type from `float` to `any` if I'm reading it correctly. Is there a reason to do this?

Reply:
It just removes the type hinting for this internal variable. Linter is happy (or can't understand the missing type) so I find it unnecessary for this "debug" type of variable. It can be `timestamps: dict[str: float] = {"start": time.time()}` ofc.